### PR TITLE
Improve man page of lepton-tragesym

### PR DESCRIPTION
**Pull request by vzh**
- Add info on source file format.
- Improve wording and formatting.
- Remove link to PCB page.

### CODE REVIEW

File: utils/docs/lepton-tragesym.1.in [hunk 1]
```diff
@@ -25,10 +26,166 @@ swap words in the pinlabel attributes if requested (only for
...
+This may be useful for large symbols like FPGAs with more than 100
+pins.
+'IP \(bu
+.B sort_labels
+.RI ( boolean )
+.br
+Sort the pins by their \*[lq]pinlabel\*[rq] attributes, which is
+useful for address ports, busses, etc.
+'IP \(bu
+.B generate_pinseq
```
Comment (graahnul-grom) [inline utils/docs/lepton-tragesym.1.in]: I would elaborate a bit the `generate_pinseq` description,
saying that numbers will be generated in sequence of pin
appearance in the `[pins] section.

File: utils/docs/lepton-tragesym.1.in [hunk 2]
```diff
@@ -39,12 +196,11 @@ scratch in Scheme by Vladimir Zhbanov <vzhbanov@gmail.com>.
 
 .SH SEE ALSO
 .BR lepton-schematic (1),
-.BR pcb (1)
 
 .SH COPYRIGHT
 .nf
 Copyright \(co 2012-2017 gEDA Contributors.
-Copyright \(co 2019 Lepton EDA Contributors.
+Copyright \(co 2019-2020 Lepton EDA Contributors.
```
Comment (graahnul-grom) [inline utils/docs/lepton-tragesym.1.in]: In other man pages we use a placeholder in copyright years
(e.g. `2019-@YEAR@`). Shouldn't it be used here?

**Comment by vzh** [comment]
@graahnul-grom,
I've added two commits to solve the issues, please take a look.

**Comment by graahnul-grom** [comment]
Очень хорошо.

**Pull request merged**